packages/next-plugin-sentry/src/on-error-client.js

```diff
@@ -1,14 +1,11 @@
-import { withScope, captureException } from '@sentry/nextjs';
-
-export default async function onErrorClient({ err, errorInfo, renderErrorProps, data, version }) {
-  // TODO: Extract some useful metadata from the router and other arguments — Kamil
+import { withScope, captureException, captureMessage } from '@sentry/nextjs';
 
+export default async function onErrorClient({ err }) {
   withScope(scope => {
-    if (typeof errorInfo?.componentStack === 'string') {
-      scope.setContext('react', {
-        componentStack: errorInfo.componentStack.trim(),
-      });
+    if (err instanceof Error) {
+      captureException(toCapture);
+    } else {
+      captureMessage(err.message);
```

Comment:
Is `err.message` always be present?

Reply:
Yes. It's an object with `message`, `name` and `statusCode`.